docs/02.API-REFERENCE.md

```diff
@@ -474,6 +474,14 @@ Possible values:
 API functions can return the `JERRY_TYPEDARRAY_INVALID` value if the
 TypedArray support is not in the engine.
 
+## jerry_versions_t
+
+Enum that contains the following elements:
+
+  - JERRY_VERSION_API_MAJOR - signals the major API version
+  - JERRY_VERSION_API_MINOR - signals the minor API version
+  - JERRY_VERSION_DEBUGGER - signals the debugger version
+  - JERRY_VERSION_SNAPSHOT - signals the snapshot version
```

Comment:
As these values are only used for one method we should add a "see also" block to refer the method which uses this.